content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,158 @@
+
+As data scientists, DVC is our toolbox. This extension turns VS Code into our
+workshop.
+
+<!-- # Why Visual Studio Code
+
+One of [our core beliefs at Iterative](https://iterative.ai/why-iterative/) is
+that our products should couple tightly with existing tools and processes for
+data science projects.
+[VS Code is the IDE of choice for many](https://insights.stackoverflow.com/survey/2021#section-most-popular-technologies-integrated-development-environment),
+including many of us at Iterative. Here are a few reasons why we really like it:
+
+VS Code...
+
+- is open source and lightweight
+- has excellent support for Git workflows
+- supports Jupyter Notebooks
+- comes with remote development and execution built-in
+- has a rich ecosystem of extensions -->
+
+# Features
+
+Our first priority was to include all DVC features related to data management
+and experiments. Here are the main features that are available when you install
+the extension right now:
+
+## DVC View: experiments, metrics, and plots
+
+The main feature of our extension is the DVC view, which provides a one-stop
+shop for everything related to your machine learning experiments. You can run
+new experiments from here and compare both metrics and plots for different
+models. Never has it been easier to analyze the results of your experiments and
+find the best model.
+
+![Experiment tables to compare different
+experiments](/uploads/images/2022-06-14/dvc-view-experiments-table.png)
+
+![Plots to visualize your
+experiments](/uploads/images/2022-06-14/dvc-view-plots.png)
+
+## Source control management
+
+On top of Git version contol for your code, the source control view now lets you
+manage datasets and models tracked by DVC. Quickly see which artifacts have been
+changed, and synchronize versions with your remote repository. Use `checkout`,
+`commit`, `add`, `push`, and `pull` straight from the interface:
+
+![Get a quick overview of changes to your data in the source control
+panel](/uploads/images/2022-06-14/source-control.png)
+
+## Command palette
+
+The DVC extension makes extensive use of the Command Palette, reducing the need
+to learn syntax by heart. Want to run a new experiment or pull data from your
+remote repository? Give the word in the Command Palette and the DVC extension
+will guide you along all required details.
+
+![Use the command palette to quickly find DVC
+features](/uploads/images/2022-06-14/command-palette.gif)
+
+<!--
+TODO: seems to overlap with Source control management; can we safely remove this?
+
+## Tracked resources
+
+Resources tracked by DVC can be viewed and managed through the tracked explorer
+view. See at a glance which datasets and models have been changed, and use
+`push` and `pull` to synchronize with your remote:
+
+![Tracked resources in the explorer
+view](/uploads/images/2022-06-14/tracked-resources.png) -->
+
+# What's next?
+
+We're excited that version 1.0 is live and available to the public! From here on
```

Comment:
We won't be releasing 1.0 for a while. Can we reword this please?

Reply:
Oops, I'd assumed the release to mark v1. Reworded.